apps/mobile/src/libs/components/components.tsx

```diff
@@ -1,5 +1,6 @@
 export { App } from "./app/app";
 export { Button } from "./button/button";
+export { LoaderWrapper } from "./common/loader-wrapper/loader-wrapper";
```

Comment:
The components in this file are already common, no need to create `common` folder